### ISSUE TITLE
有關預測和訓練時 輸入句數（batch size)不同的問題

### ISSUE THREAD

**Issue by billy800413** [NELSONZHAO/zhihu#8]
請問一下有辦法能讓預測時真的只要輸入一個句子就好嗎？
就是這段程式碼不用*batch_size
    answer_logits = sess.run(logits, {input_data: [text]*batch_size, 
                                      target_sequence_length: [len(input_word)]*batch_size, 
                                      source_sequence_length: [len(input_word)]*batch_size})[0]
